lib/chef/resource.rb

```diff
@@ -1382,9 +1385,16 @@ def self.action(action, &recipe_block)
     #   created resource with its identity values filled in.
     #
     def self.load_current_value(&load_block)
+      include LoadCurrentValueDSL
```

Comment:
Agreed - I am curious why we don't just include it at the top level (or just define the method in the class since it looks like thats all the module is doing)